app/models/nic/base.rb

```diff
@@ -22,7 +22,7 @@ class Base < ActiveRecord::Base
     validates :mac, :uniqueness => {:scope => :virtual},
               :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? && !nic.virtual? }, :allow_blank => true
```

Comment:
I think this is quite unrelated of my commit in this PR, anyway I reported it as http://projects.theforeman.org/issues/10442 and I'll send separate PR